BackEndApp/v1/views.py

```diff
@@ -37,7 +39,9 @@ def generate(request):
     project_id = request.data.get("project_id")
     store_obj = Store(user)
     if not store_obj.exist(project_id):
-        raise Exception("No such project exists")
+        error_message = "No such project exists"
+        logger.error(f"[API][POST][generate] {error_message}")
```

Comment:
```
[API][POST][generate]
```
Is this supposed to resolve to some string?

Reply:
Nope. It is a more structured way of logging. This will help in debugging (ex: filtering logs in the shell using grep).

Reply:
![image](https://user-images.githubusercontent.com/35848285/136150681-13a6c660-0e82-4d7d-8361-8fd365e4ed0c.png)
Could rather have a message like the above?

Reply:
@prajwalnayak7